Scripts/extend.gd

```diff
@@ -0,0 +1,13 @@
+class_name extend
+
+# Extend class:
+# a class containing static functions for global use.
+
+#Gets all of the parents children (grand children, great grand children, ect)
+static func get_all_Children(root_node) -> Array[Node]:
```

Comment:
If this is simply a func you want to be globally available, I would recommend making the script a global autoload singleton instead of a static func. Will cause way less issues. Furthermore, this function is (at least in my experience) not critical enough to be made global.

Reply:
actually I was going to have this as a sort of S.O.L class to add random methods that are commonly used, like books to integers for flipping,
or a change scene function. this was mainly an implementation for hard core programmers instead of developers, so adding it as a global class wouldn't be preforment or efficient as it would only stand to cause confusion. 

the reason that get all children should be in there is due to potential inventory, procedural, or basic implementation.

Reply:
utility Functions does sound better than Extend